apps/nextjs/sentry.common.config.ts

```diff
@@ -1,7 +1,7 @@
 import { env } from "@/env";
 
 export const commonConfig = {
-  dsn: "https://84640ca2ac12acbfd7a06e0a02e3ba56@o4508048702767104.ingest.us.sentry.io/4508049493196800",
+  dsn: process.env.NEXT_PUBLIC_SENTRY_DSN,
```

Comment:
```suggestion
  dsn: env.NEXT_PUBLIC_SENTRY_DSN,
```

(this will probably get flagged by the linter)